openedx/adg/lms/branding_extension/constants.py

```diff
@@ -1,5 +1,9 @@
 """
 Constants for branding extension
 """
+from django.conf import settings
+
 TARGET_BLANK = '_blank'
 TARGET_SELF = '_self'
+
+FAQs_LINK = f'{settings.SUPPORT_LINK}/hc/en-us/sections/360010666551-FAQ'
```

Comment:
Are we sure this link will never change?

Reply:
There could be Arabic link later on but we decided to work with only link for now.